backend/alembic/versions/cd2cdb335e90_db_migrate.py

```
@@ -1,33 +1,34 @@
+"""db migrate
 
-Revision ID: 0ffdc7e11ff6
-Revises: 6bdd563d49e1
-Create Date: 2025-03-17 03:04:09.672404
+Revision ID: cd2cdb335e90
+Revises: 4bd138c7e55b
+Create Date: 2025-04-29 03:01:00.542644
 
 """
-
 from typing import Sequence, Union
```

Ruff (0.8.2)

Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)